logistics_planning_invoicing/models/account_move_line.py

```diff
@@ -0,0 +1,28 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models, fields
+
+
+class AccountMoveLine(models.Model):
+    _inherit = 'account.move.line'
+
+    logistics_schedule_id = fields.Many2one('logistics.schedule')
+
+    def create(self, values):
+        res = super().create(values)
+        for record in res.filtered(lambda x: x.logistics_schedule_id):
+            record.logistics_schedule_id.account_move_line_id = record.id
+
+    def write(self, values):
+        for record in self:
+            logistics_schedule_id = values.get('logistics_schedule_id')
+
+            if record.logistics_schedule_id and logistics_schedule_id is not None:
+                record.logistics_schedule_id.account_move_line_id = False
+
+            if logistics_schedule_id:
+                ls_id = record.env['logistics.schedule'].browse(logistics_schedule_id)
+                ls_id.account_move_line_id = record
+
+        return super().write(values)
```

Comment:
This method could be improved, because values is the same for every iteration, so obtaining `values.get('logistics_schedule_id')` could be made once at the beginning of the method.

Indeed, we should skip the whole process if is not passed and consider self as a single record, because we cannot update several account move lines with the same schedule (1 schedule => 1 invoice line), not tested:
```suggestion
        if "logistics_schedule_id" in values:
            logistics_schedule_id = (
                values.get("logistics_schedule_id")
                and self.env["logistics.schedule"].browse(values.get("logistics_schedule_id"))
                or False
            )
            if logistics_schedule_id:
                logistics_schedule_id.account_move_line_id = self
            else:
                self.logistics_schedule_id.account_move_line_id = False
        return super().write(values)
```